model/account.go

```diff
@@ -0,0 +1,10 @@
+package model
+
+type Account struct {
+	ID       uint   `gorm:"primary_key"`
+	User     string `gorm:"not null;unique"`
+	Password string `gorm:"not null"`
+	Sex      string `gorm:"type:char(1)"`
+	Email    string `gorm:"unique"`
```

Comment:
I'd let this decision be made by server admin rather then developers. We should not enforce this at DB level.